__TESTS__/TestConstants/createEntryMockedFS.ts

```diff
@@ -12,6 +12,11 @@ const TEST_FILE_STRUCTURE = {
           'Border.d.ts' : '// Some content'
         }
       },
+      constants: {
```

Comment:
Note that this is only needed if you're writing a test (as this is a MOCKED file system).
We can keep it here, but just keep that in mind